yb-voyager/cmd/getDataMigrationReportCommand.go

```diff
@@ -418,6 +418,7 @@ func init() {
 	registerExportDirFlag(getDataMigrationReportCmd)
 	getDataMigrationReportCmd.PersistentFlags().StringVarP(&config.LogLevel, "log-level", "l", "info",
 		"log level for yb-voyager. Accepted values: (trace, debug, info, warn, error, fatal, panic)")
+	getDataMigrationReportCmd.PersistentFlags().MarkHidden("log-level")
```

Comment:
just curious, if we have defined it on root why do we need on bulk and get command here.

Reply:
Something I wanted to also discuss! - these commands do not call the `registerCommonGlobalFlags` function. Not ideal; we should pick it up sometime.

Reply:
I think it was intentional that some global flags are not required for these commands e.g. get datamigration one doesn't need the yes or send-diagnostics etc..